extensions/wikia/Email/EmailController.class.php

```diff
@@ -135,10 +154,21 @@ protected function setErrorResponse( ControllerException $e ) {
 	}
 
 	protected function getTargetLang() {
+		if ( !empty( $this->targetLang ) ) {
+			return $this->targetLang;
+		}
```

Comment:
Maybe now that we have ```$this->targetLang```, we should assign it to the default if we get here.  We call ```getTargetLang``` quite a bit.